lib/schemas/Sass.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "kendo-cli://schemas/Sass.json",
+    "title": "Sass schemas",
+
+    "definitions": {
+
+        "variableType": {
+            "description": "Valid Sass variable type",
+            "markdownDescription": "Valid Sass variable type. See [Sass values](https://sass-lang.com/documentation/values)",
+            "type": "string",
+            "enum": [
+                "color",
+                "boolean"
```

Comment:
Why did you add only these two types? How about `number`, `string`, etc?

For example, in the current ThemeBuilder - `Border radius` has `variableType` - `number`.